how-to-make-a-pitch-into-a-blogpost/readme.md

```diff
@@ -0,0 +1,51 @@
+>**Goal of this guide:** Explaining how to divide the process from idea to end result into small workable steps that are all individually publishable articles.
+
+# How to turn a pitch into a publishable article
+
+In this guide I will try to explain how to divide the process from idea to end result into small workable steps that are all individually publishable articles.
```

Comment:
Consider add more value to this because it's a bit tiring to read the same as in the **Goal** part at the top